docs/projects/template-fps/setup-and-installing.md

```diff
@@ -1 +1,27 @@
-// TODO: Populate this page from content from: https://github.com/improbable/ECS-FPS/blob/master/docs/setup-and-installing.md
+# Set up the FPS Starter Project
+
+1. Follow the [machine setup guide]({{urlRoot}}/setup-and-installing#set-up-your-machine).
```

Comment:
would remove the word machine. it's just a setup guide

Reply:
Hmmm maybe - I want to avoid them going through the rest of the guide